src/Hazelcast.Net.Testing/SingleMemberClientRemoteTestBase.cs

```diff
@@ -40,7 +40,7 @@ public async Task ClientOneTimeTearDown()
         /// <summary>
         /// Gets the Hazelcast client.
         /// </summary>
-        public IHazelcastClient Client { get; private set; }
+        public static IHazelcastClient Client { get; private set; }
```

Comment:
Not sure I like this. Anything static in test classes and especially in test *base* classes can lead to collisions between tests. Can it remain an instance variable?

Reply:
ok